scylladb/sphinx-scylladb-theme
# Reverse branch order

Closes #154

Before: 

![image](https://user-images.githubusercontent.com/9107969/118115594-110cd800-b3e1-11eb-8723-82a26fe34b84.png)

After:

![image](https://user-images.githubusercontent.com/9107969/118115683-2e41a680-b3e1-11eb-8c9f-ba5d4ef6fa29.png)

## How to test this PR

In ``conf.py``, replace ``BRANCHES = ['master']`` with ``BRANCHES = ['master', 'new-design']`` to build docs for multiple versions.

You should see new-design "listed before master:

![image](https://user-images.githubusercontent.com/9107969/118650859-d2aa5b00-b7dc-11eb-8ee1-379ff2f39c89.png)
